.travis.yml

```diff
@@ -20,5 +17,3 @@ script:
   - cargo build
   - cd ..
   - cargo doc
```

Comment:
I'm thinking we might want to remove this line as well, otherwise that dance with checking out everything from git would have to be applied to the current crate as well